app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    _, file, new_file = command.split()
+
+    if file != new_file:
+        with open(file, "r") as f_in, open(new_file, "w") as f_out:
```

Comment:
`file_in`, `file_out` are not that long names, but they are much more descriptive

Reply:
Done